src/collections/mod.rs

```diff
@@ -1,6 +1,4 @@
-//! This module contains the parallel iterator types for standard
-//! collections. You will rarely need to interact with it directly
-//! unless you have need to name one of the iterator types.
+//! Parallel iterator types for [standard collections](std::collections).
```

Comment:
I think this style of auto-link only works with nightly.  Can you manually link it?